test/arithmetic/addend_compressor_test.dart

```diff
@@ -8,13 +8,56 @@
 // Author: Desmond Kirkpatrick <desmond.a.kirkpatrick@intel.com>
 
 import 'dart:async';
+import 'dart:math';
 import 'package:rohd/rohd.dart';
 import 'package:rohd_hcl/rohd_hcl.dart';
 import 'package:rohd_hcl/src/arithmetic/evaluate_compressor.dart';
 import 'package:rohd_hcl/src/arithmetic/evaluate_partial_product.dart';
 import 'package:rohd_hcl/src/arithmetic/partial_product_sign_extend.dart';
 import 'package:test/test.dart';
 
+/// This extension will eventually move to ROHD once it is proven useful
+extension LogicValueMajority on LogicValue {
+  /// Compute the unary majority on LogicValue
+  bool majority() {
+    if (!isValid) {
+      return false;
+    }
+    final zero = LogicValue.filled(width, LogicValue.zero);
+    var shiftedValue = this;
+    var result = 0;
+    while (shiftedValue != zero) {
+      result += (shiftedValue[0] & LogicValue.one == LogicValue.one) ? 1 : 0;
+      shiftedValue >>>= 1;
+    }
+    return result > (width ~/ 2);
+  }
+
+  /// Compute the first One find operation on LogicValue, returning its position
+  int? firstOne() {
```

Comment:
you could still import that other non-public file to gain access to this, rather than copy-paste the function

Reply:
i still see it in both "evaluate_partial_product.dart" and "logicvalue_extension.dart"